scalding-core/src/main/scala/com/twitter/scalding/FileSource.scala

```diff
@@ -196,11 +214,20 @@ abstract class FileSource extends SchemedSource with LocalSourceOverride {
         }
       }
 
-      case Local(_) => {
-        val file = new java.io.File(localPath)
-        if (!file.exists)
+      case Local(strict) => {
+        val files = localPaths.map{
```

Comment:
let's use: `localPaths.map { p => new java.io.File(p) }` to be more consistent with scalding's style.